pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -2,26 +2,28 @@
 
 ### Deposit to Subaccount
 
+The below cmd allows depositing funds into a subaccount (`x/subaccounts`) from a sender's `x/bank` account. 
+Typically, this cmd is used to self deposit, which can be done by setting the sender and recipient address to be the same address, as in the provided example.
 ```bash
 dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
 ```
 
 Example:
 ```bash
-NATIVE_TOKEN_DENOM=adv4tnt
-dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000$NATIVE_TOKEN_DENOM
+dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000000000000000<native_token_denom>
 ```
-Note: Token denoms for transfer amounts and fees can be found [here](https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom)
+Note: Token denoms for transfer amounts and fees can be found [here](../networks/network1/network_constants#native-token-denom)
```

Comment:
Given that this applies to both deposit and withdrawal, would it make sense to move this to line 2?